src/AppInstallerRepositoryCore/Microsoft/Schema/1_0/ManifestTable.cpp

```diff
@@ -83,17 +83,25 @@ namespace AppInstaller::Repository::Microsoft::Schema::V1_0
         SQLite::Statement ManifestTableGetValuesById_Statement(
             const SQLite::Connection& connection,
             SQLite::rowid_t id,
-            std::initializer_list<SQLite::Builder::QualifiedColumn> columns)
+            std::initializer_list<SQLite::Builder::QualifiedColumn> columns,
+            std::initializer_list<std::string_view> manifestColumnNames)
         {
+            THROW_HR_IF(E_UNEXPECTED, manifestColumnNames.size() != 0 && manifestColumnNames.size() != columns.size());

```

Comment:
```suggestion
            THROW_HR_IF(E_UNEXPECTED, manifestColumnNames.size() != columns.size());
```
I don't think this code works if `manifestColumnNames` is not provided along with `columns`.